### ISSUE TITLE
Does vSwarm support Firecracker-Containerd

### ISSUE THREAD

**Issue by DanielLee343** [vhive-serverless/vSwarm#347]
Hi, I noticed that vSwarm only supports applications deployed and invoked by Knative, is that true? Do we have a feature to run those functions via Firecracker-containerd? It's interesting to see how real-world FaaS applications perform snapshots. Or do you have any advise to walk around this? Thanks!

**Comment by ustiugov** [comment]
@DanielLee343 yes, all vHive runtimes are supported (but only container-based setting is tested in CI). However, to use Firecracker, you need to modify YAML files, which is the limitation we are currently working on. 

More details: https://github.com/vhive-serverless/vHive/blob/main/docs/quickstart_guide.md#iv-deploying-and-invoking-functions-in-vhive
